vcd/config.go

```diff
@@ -424,10 +432,15 @@ func (c *Config) Client() (*VCDClient, error) {
 		return nil, fmt.Errorf("something went wrong while retrieving URL: %s", err)
 	}
 
+	userAgent := fmt.Sprintf("terraform-provider-vcd/%s (%s/%s; isProvider:%t)",
+		BuildVersion, runtime.GOOS, runtime.GOARCH, strings.ToLower(c.SysOrg) == "system")
```

Comment:
I see the User-Agent with the new build:

Old build (grep):
```
2020/10/02 09:59:45 	Vary: [Accept-Encoding, User-Agent]
2020/10/02 09:59:46 	Vary: [Accept-Encoding, User-Agent]
...
```

New build (grep):
```
2020/10/05 11:52:30 	User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
2020/10/05 11:52:31 	Vary: [Accept-Encoding, User-Agent]
2020/10/05 11:52:31 	User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
2020/10/05 11:52:32 	User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
2020/10/05 11:52:32 	User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
2020/10/05 11:52:33 	Vary: [Accept-Encoding, User-Agent]
2020/10/05 11:52:33 	User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
2020/10/05 11:52:33 	Vary: [Accept-Encoding, User-Agent]
...
```
Looks great in the request:
```
2020/10/05 11:52:30 Req header:
2020/10/05 11:52:30     User-Agent: [terraform-provider-vcd/v2.9.0-28-g5b5d93d (darwin/amd64; isProvider:true)]
```
Is it expected that in the response we have it empty?
```
2020/10/05 11:52:31 Response header:
2020/10/05 11:52:31     X-Vmware-Vcloud-Request-Execution-Time: [49]
2020/10/05 11:52:31     Cache-Control: [no-store, must-revalidate]
2020/10/05 11:52:31     Vary: [Accept-Encoding, User-Agent]
```

Reply:
> Is it expected that in the response we have it empty?

Yes. That is server sending data back (server is not expected to have/send User-Agent)